packages/backend/src/controllers/authentication/strategies/googleStrategy.ts

```diff
@@ -62,7 +62,21 @@ export const googlePassportStrategy: GoogleStrategy | undefined = !(
                           issuerType: OpenIdIdentityIssuerType.GOOGLE,
                       },
                   };
+                  const hasBigqueryScope = params.scope.includes('bigquery');
 
+                  if (hasBigqueryScope) {
```

Comment:
this also means, if you created the project using SSO , and enable requiredCredentials, you will not need to authenticate again.